src/streaming/utils/TTMLParser.js

```diff
@@ -127,10 +177,13 @@ function TTMLParser() {
 
         eventBus.trigger(Events.TTML_TO_PARSE, content);
 
-        const imsc1doc = fromXML(content.data, function (msg) {
+        let imsc1doc = fromXML(content.data, function (msg) {
             errorMsg = msg;
         }, metadataHandler);
 
+        // TODO: Fix bug in imscJS V1.1.4 - Can be removed when v1.1.5 is released and used
```

Comment:
Any estimation when 1.1.5 will be released? Would be nice to use 1.1.5 instead of adding a workaround in dash.js

Reply:
That would be a question for @palemieux.

Reply:
What is the bug in imscJS?

Reply:
It was sandflow/imscJS#250, now fixed on the default branch of imscJS.

Reply:
Ok. We can do another quick point release. What is your schedule? Any other pressing issue?

Reply:
Thanks, we are aiming for a new dash.js release end of this month. I am not aware of any other issue, but @nigelmegitt and @mattjuggins can probably provide more information on that.

Reply:
> Any other pressing issue?

@palemieux I've had a report that on some devices running chrome the fillLineGap functionality isn't working correctly, and 1 pixel gaps are appearing between the background areas on adjacent lines. Not sure if there's an issue for it yet, and I'm unable to reproduce the behaviour myself (I don't have the right device), but this would be an imscJS issue almost certainly. It may not be super urgent.

Reply:
I have set a reminder to make a release on the 19.

Reply:
Perfect. I'll remove this and update the imscJS dependency when this is released.

Reply:
> Perfect. I'll remove this and update the imscJS dependency when this is released.

1.1.5-rc.1 at https://www.npmjs.com/package/imsc/v/1.1.5-rc.1

1.1.5 is scheduled for next Friday.

Reply:
> I've had a report that on some devices running chrome the fillLineGap functionality isn't working correctly, and 1 pixel gaps are appearing between the background areas on adjacent lines.

Raised by @pjcherriman as sandflow/imscJS#253

Reply:
Released at https://github.com/sandflow/imscJS/releases/tag/1.1.5

Reply:
I've updated the imscJS version and removed this workaround.